salt/core/systemd/init.sls

```diff
@@ -1,7 +1,19 @@
 # Stop RemoveIPC killing all processes by a user when they log out.
+{% if grains['osrelease']|float >= 20.04 %}
+/etc/systemd/logind.conf.d/customize.conf:
+  file.managed:
+    - source: salt://core/systemd/files/logind.conf
+    - makedirs: True
+    - watch_in:
+      - service: systemd-logind
 
+{% else %}
```

Comment:
Similar to an earlier comment, I believe Ubuntu 18 already checks /etc/systemd/logind.conf.d/*.conf (based on running `man logind.conf`), so it'd be nice to use the same configuration across versions.

Reply:
Done - using same new configuration across versions.